js/createentry.js

```diff
@@ -0,0 +1,155 @@
+const apiURL = "http://localhost:";
```

Comment:
what if apiURL was "http://localhost:8002" rather than just localhost:?
i am thinking of a way for automated deployment and in deployment it wont be :8002 but i can do a find and replace on the project to replace the localhost:8002

Reply:
because the servers run on different ports 8001 and 8002

Reply:
but if can resolve it and make them both on the same port then also can

Reply:
what about 

accountsMicroserviceURL, authMicroserviceURL...

Reply:
so you want separate urls for each service is it

Reply:
yeah since that will help deployment a lot

Reply:
ok sure i will change it